src/templates/static/legal/tac.jsx

```diff
@@ -25,29 +25,29 @@ const Tac = () => (
                         text   : it.L('Terms and Conditions'),
                         submenu: [
                             { id: 'legal-binary', text: it.L('Binary Options') },
-                            { id: 'legal-mt',     text: it.L('MetaTrader') },
+                            { id: 'legal-mt', text: it.L('MetaTrader') },
                         ],
                     },
-                    { id: 'mam',              text: it.L('MAM'), className: 'invisible' }, // toEnableMAM: remove className
```

Comment:
why is the space removed? they were being aligned by properties

Reply:
Fixed. It was an auto-linting thing that wasn't configured right